### PR TITLE
Habitat_confirmations spreadsheet

### PR DESCRIPTION
**Pull request by newgraph-lschick**
- `inputs_extracted.R` now updates sheet 2 of `habitat_confirmations.xls` with all data except some data which was still added by hand. 
- Data added by hand can be found here https://github.com/NewGraphEnvironment/fish_passage_skeena_2023_reporting/issues/64#issuecomment-2075284197
- Some data is still missing and it noted here https://github.com/NewGraphEnvironment/fish_passage_skeena_2023_reporting/issues/64#issuecomment-2075281062.

### CODE REVIEW

File: data/backup/step_1_ref_and_loc_info.csv
Comment (NewGraphEnvironment) [inline data/backup/step_1_ref_and_loc_info.csv:1]: fyi - the UTM method in the skeena 2022 repo was incorrect.  peace 2022 however was correct.  Our method is most commonly GPS General.  `GIS` is when you derive the points in something like Q. although we do that sometimes (ex. move points) - it is almost always based on our `GPS` unit points and tracks

File: scripts/01_prep_inputs/0100-extract-inputs.R [hunk 1]
```diff
@@ -755,6 +756,9 @@ plot_fish_hist
 
 
 ####-----------summary tables for input to spreadsheet----------------------
+dir_gis <- 'sern_skeena_2023'
```
Comment (NewGraphEnvironment) [inline scripts/01_prep_inputs/0100-extract-inputs.R:759]: this is cool

File: scripts/01_prep_inputs/0100-extract-inputs.R [hunk 2]
```diff
@@ -793,17 +795,53 @@ hab_fish_input <- left_join(
...
+  # Add in make and model of ef
+  mutate(model = case_when(local_name %like% 'ef' ~ 'halltech HT2000'),
+         make = case_when(local_name %like% 'ef' ~ 'other'),
+         method_number = "1")
+
+## Burn to a csv so you can cut and paste into your fish submission
+## The following fields need to be added by hand from the fish cards: sampling_method, ef_seconds, enclosure, voltage, frequency
 hab_fish_input %>%
-  readr::write_csv(file = paste0(getwd(), '/data/inputs_extracted/hab_con_fish_summary.csv'),
+  readr::write_csv(file = paste0('data/inputs_extracted/hab_con_fish_summary.csv'),
```
Comment (NewGraphEnvironment) [inline scripts/01_prep_inputs/0100-extract-inputs.R:844]: no need for paste0 once we ditch getwd

**Review by NewGraphEnvironment — APPROVED** [review]
thank you!

File: data/backup/step_1_ref_and_loc_info.csv
Comment (newgraph-lschick) [reply data/backup/step_1_ref_and_loc_info.csv:1]: okay. Let me know if you want me to correct `UTM method` or if you will do that in next PR?

**Comment by newgraph-lschick** [comment]
Thank you!

**Pull request merged**